docs/index.rst

```diff
@@ -28,19 +28,62 @@
+simple to manage and orchestrate your dbt workflows using `Apache Airflow® <https://airflow.apache.org/>`_, saving you
+time and effort. By automatically turning dbt workflows into Airflow DAGs, Cosmos allows you to focus on building
+high-quality data models without the hassle of managing complex integrations.
 
+To get started right away, please check out our `Quickstart Guides <https://astronomer.github.io/astronomer-cosmos/getting_started/index.html>`_.
+You can also explore more examples in the `/dev/dags <https://github.com/astronomer/astronomer-cosmos/tree/main/dev/dags>`_
+or in the `cosmos-demo repo <https://github.com/astronomer/cosmos-demo>`_.
 
-Example Usage
-___________________
+To learn more about about Cosmos, please read on.
 
-You can render a Cosmos Airflow DAG using the ``DbtDag`` class. Here's an example with the `jaffle_shop project <https://github.com/dbt-labs/jaffle_shop>`_:
+
+What Is Astronomer Cosmos?
+___________________________
+
+Astronomer Cosmos is an open-source library that bridges Apache Airflow and dbt, allowing you to easily transform your
+dbt projects into Airflow DAGs and manage everything seamlessly. With Cosmos, you can write your data transformations
+using dbt and then schedule and orchestrate them with Airflow, making the entire process smooth and straightforward.
+
+**Why Cosmos?**  Integrating dbt and Airflow can be complex, but Cosmos simplifies it by seamlessly connecting these
+powerful tools—letting you focus on what matters most: delivering impactful data models and results without getting
+bogged down by technical challenges.
+
+
+Why Should You Use Cosmos?
+___________________________
+
+Cosmos makes orchestrating dbt workflows:
+
+- **Effortless**: Transform your dbt projects into Airflow DAGs without writing extra code—Cosmos handles the heavy lifting.
+- **Reliable**: Rely on Airflow's robust scheduling and monitoring features to ensure your dbt workflows run smoothly and efficiently.
+- **Scalable**: Easily scale your workflows to match growing data demands, thanks to Airflow's distributed capabilities.
+
+Whether you're handling intricate data tasks or looking to streamline your processes, Cosmos helps you orchestrate dbt
+with Airflow effortlessly, saving you time and letting you focus on what truly matters—creating impactful insights.
+
+
+Example Usage: Jaffle Shop Project
+__________________________________
+
+Let's explore a practical example to see how Cosmos can convert the dbt workflow into an Airflow DAG.
+
+The `jaffle_shop project <https://github.com/dbt-labs/jaffle_shop>`_ is a sample dbt project that simulates an e-commerce store's data.
+The project includes a series of dbt models that transform raw data into structured tables, such as sales, customers, and products.
+
+Below, you can see what the original dbt workflow looks like in a lineage graph. This graph helps illustrate the
+relationships between different models:
+
+.. image:: /_static/jaffle_shop_dbt_graph.png
```

Comment:
```suggestion
.. image:: https://raw.githubusercontent.com/astronomer/astronomer-cosmos/main/docs/_static/jaffle_shop_dbt_graph.png
```
wydt?

Reply:
When I tried that, it didn't render (I assume because it wasn't in version control yet). What's the difference between this, and using the relative path so that it works locally?

Reply:
I think they’re just alternatives to each other. However, I would consider `/_static/jaffle_shop_dbt_graph.png` to be an absolute path, and I’m unsure if that would work. I’m wondering if the correct relative path should be `./_static/jaffle_shop_dbt_graph.png` instead.

Reply:
When I initially tried `https://raw.githubusercontent.com/astronomer/astronomer-cosmos/main/docs/_static/jaffle_shop_dbt_graph.png`, I received an error. Intuitively that made sense since the image is new, and does not exist on the main branch. 

I then tried `./_static/jaffle_shop_dbt_graph.png`, and received the same error. Last attempt was `/_static/jaffle_shop_dbt_graph.png`, and the docs compiled properly. 

As written, the image shows up in the PR preview environment.  See [here](https://deploy-preview-1251--sunny-pastelito-5ecb04.netlify.app) - it's the image for the dbt lineage graph. I'm fine with switching to pulling the image from github main branch if that is what you want, but it means that this PR won't work until after the PR has been merged to main.

Reply:
hmm, for some reason, I saw the image is not rendering 
<img width="1568" alt="Screenshot 2024-10-18 at 12 30 14 AM" src="https://github.com/user-attachments/assets/72560283-89de-413b-b953-277bdfd3965d">

I case of relative path it may not render on PyPi

Reply:
Ah yes I see the disconnect. It's not going to render in github. But as written, it does render in the docs themselves when they are generated (which is the most important). When I switch it, it renders in github, but not in the docs.